templates/pages/admin/user.substitute.html.twig

```diff
@@ -0,0 +1,211 @@
+ # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ #
+ # ---------------------------------------------------------------------
+ #}
+{% import 'components/form/fields_macros.html.twig' as fields %}
+
+{% set bg = '' %}
+{% if item.isDeleted() %}
+   {% set bg = 'asset-deleted' %}
+{% endif %}
+
+<div class="asset {{ bg }}">
+
+{# Form Header #}
+{% set target       = params['target'] ?? item.getFormURL() %}
+{% set canedit      = params['canedit'] ?? true %}
+{% set withtemplate = params['withtemplate'] ?? '' %}
+{% set rand         = random() %}
+{% set nametype     = params['formtitle'] ?? item.getTypeName(1) %}
+{% set friendlyname = params['friendlyname'] ?? item.getHeaderName() %}
+{% set no_id        = params['noid'] ?? false %}
+{% set id           = item.fields['id'] ?? -1 %}
+{% set formoptions  = params['formoptions'] ?? '' %}
+
+{% set entity_id = 0 %}
+{% set entity_name = '' %}
+{% if item.isEntityAssign() %}
+   {% if item.getType() == 'Entity' and item.fields['id'] == 0 %}
+      {% set entity_id = null %}
+   {% else %}
+      {% set entity_id = params['entities_id'] ?? item.getEntityID() ?? session('glpiactive_entity') %}
+   {% endif %}
+
+   {% if is_multi_entities_mode() %}
+      {% set entity_name = get_item_name('Entity', item.getEntityID()) %}
+   {% endif %}
+{% endif %}
+
+{% if item.canEdit(user.fields['id']) %}
+   <form name="massaction_{{ rand }}" id="massaction_{{ rand }}" method="post"
+         action="{{ path('/front/massiveaction.php') }}" data-submit-once>
+      <div id="massive_container_{{ rand }}"></div>
+      <input type="hidden" name="_glpi_csrf_token" value="{{ csrf_token() }}" />
+   </form>
+
+   <form name="asset_form" method="post" action="{{ target }}" {{ formoptions|raw }} enctype="multipart/form-data" data-submit-once>
+      <input type="hidden" name="entities_id" value="{{ entity_id }}" />
+      {% if _request['_in_modal'] is defined and _request['_in_modal'] == "1" %}
+         <input type="hidden" name="_in_modal" value="1"/>
+      {% endif %}
+{% endif %}
+
+   <div id="mainformtable">
+      {% set template_name = item.fields['template_name']|verbatim_value %}
+      {% if withtemplate == 2 and not item.isNewItem() %}
+         <input type="hidden" name="template_name" value="{{ template_name }}" />
+         {% set nametype = __('Created from the template %s')|format(template_name) %}
+      {% elseif withtemplate == 1 %}
+         <input type="hidden" name="is_template" value="1" />
+      {% elseif item.isNewItem() %}
+         {% set nametype = __('%1$s - %2$s')|format(__('New item'), nametype) %}
+      {% else %}
+         {% if noid == false and (session('glpiis_ids_visible') or nametype|length == 0) %}
+            {% set nametype = __('%1$s - %2$s')|format(nametype, item.fields['id']) %}
+         {% endif %}
+      {% endif %}
+
+      {{ call_plugin_hook(constant('Glpi\\Plugin\\Hooks::PRE_ITEM_FORM'), {'item': item, 'options': params}) }}
```

Comment:
Not sure how it should be handled. This hook call should probably be removed.

Reply:
after our discussion, I keep it for now. Let's see other reviewers comment.